### PR TITLE
Prepare for Docker Hub release

### PR DESCRIPTION
**Pull request by sirn**
* Use s6-overlay for init in Dockerfile.
* Build assets and immediately discard node_modules to reduce file sizes.
* Remove Habitus and building artifacts from when we were private.
* Use `mix omg.server --no-watch`
* Switch to Docker Hub.

### CODE REVIEW

**Comment by sirn** [comment]
I forgot to update README.

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Public
+/public/uploads
+!/public/uploads/test.txt
+
+# Dev
+/.git
+/.github
+.env
+.DS_Store
+Thumbs.db
```
Comment (unnawut) [inline .dockerignore]: Should we also ignore `.elixir_ls`? (generated by a popular VSCode plugin for Elixir, some of us are using VSCode now)

**Comment by unnawut** [comment]
Oh wait... this is not ready for review right? Was mistaken when I saw "review required" in the PR list 🤦‍♂️

File: README.md [hunk 2]
```diff
@@ -92,6 +92,25 @@ The demo server applications have been deployed and are available at the followi
...
 The following section will get you up to speed on the eWallet and show you how to deploy it in local.
 
+## Using Docker image
+
+We provides a Docker image for quick installation at the [omisego/ewallet](https://hub.docker.com/r/omisego/ewallet/) repository on Docker Hub with the following tags:
+
+* `latest` tracks current (i.e. develop branch)
+* `stable` tracks stable (i.e. master branch)
+
+Additionally, all commits in develop and master branches are also built. It is highly recommend to pin the version to certain commit in anything that resembles production environment. An easiest way to get the Docker image running would be (assuming [PostgreSQL]() is already setup):
```
Comment (T-Dnzt) [inline README.md]: `highly recommended` - `to certain commits` (`specific` might be more appropriate than `certain`)

Comment (T-Dnzt) [reply README.md]: Link for PostgreSQL missing.

File: apps/admin_api/test/admin_api/v1/controllers/self_controller_test.exs [hunk 3]
```diff
@@ -1,8 +1,8 @@
 defmodule AdminAPI.V1.SelfControllerTest do
   use AdminAPI.ConnCase, async: true
-  alias EWalletDB.Account
+  alias EWallet.Helper
```
Comment (T-Dnzt) [inline apps/admin_api/test/admin_api/v1/controllers/self_controller_test.exs]: Where are those changes coming from? `develop`? Was this branch of from `develop`? In that case, we might want to merge it there first?

Comment (sirn) [reply apps/admin_api/test/admin_api/v1/controllers/self_controller_test.exs]: Welp, wrong base

File: README.md (hunk 2 above)
Comment (sirn) [reply README.md]: 👌

**Pull request merged**